src/common/snippets/include/snippets/op/loop.hpp

```diff
@@ -95,17 +91,27 @@ class LoopEnd : public LoopBase {
     void set_work_amount(size_t new_work_amount);
     void set_increment(size_t new_increment);
     void set_evaluate_once(bool once);
+    void set_work_with_buffer(bool buffer);
     // Used to propagate information about Loop structure, needed to simplify some optimizations. For example,
     // to skip pointer increments when outer Loop is empty, and work_amount == vector_size (one inner vector Loop)
     // true by default, the optimizations enabled if it's false;
     bool has_outer_loop;
```

Comment:
`has_outer_loop` is not initialized in the default constructor
```suggestion
    bool has_outer_loop = true;
```

Reply:
Done, thanks